app/services/userService.js

```diff
@@ -12,3 +12,26 @@ const getOne = email => {
 exports.getByEmail = email => {
   return getOne({ email });
 };
+
+exports.createUserAdmin = user => {
+  return User.findOrCreate({
+    where: { email: user.email },
+    defaults: { name: user.name, surname: user.surname, password: user.password, isAdmin: true }
+  })
+    .then(([admin, isNewUser]) => {
+      if (!isNewUser) {
+        return admin.update({ isAdmin: true });
+      }
+      return admin;
+    })
+    .catch(err => {
+      throw errors.databaseError;
+    });
+};
+
+exports.update = (isAdmin, user) => {
```

Comment:
This functions isn't been used anywhere.
And also, you are expecting to receive the user as a param, you could do `user.update(...)` instead of using the `User` model.